checkstyle-sonar-plugin/pom.xml

```diff
@@ -49,7 +42,7 @@
     <dependency>
       <groupId>org.sonarsource.sslr-squid-bridge</groupId>
       <artifactId>sslr-squid-bridge</artifactId>
-      <version>2.6.1</version>
+      <version>2.7.0.377</version>
```

Comment:
sslr-squid-bridge user sonar version as 6.7
https://github.com/SonarSource/sslr-squid-bridge/blob/2.7.0.377/pom.xml#L51

so we will use the same sonar version for testing and in CI.